content/kubermatic/master/tutorials_howtos/how_to_contribute_to_kkp/_index.en.md

```diff
@@ -0,0 +1,63 @@
++++
+title = "How To Contribute to Kubermatic Kubernetes Platform (KKP)"
+date = 2021-04-24T15:00:15+02:00
+weight = 25
++++
+
+
+## Contributing to KKP
+
+Kubermatic Kubernetes Platform (KKP) is an open-source project to centrally manage the global automation of thousands of Kubernetes clusters across multi-cloud, on-prem and edge with unparalleled density and resilience. Therefore, contributors in different areas but not limited to project automation, code review, tooling, documentation, testing etc., for the holistic improvement of the KKP project is highly welcomed. Kubermatic projects are [Apache 2.0 licensed](https://github.com/kubermatic/kubermatic/blob/master/LICENSE) and accept contributions via GitHub pull requests from interested contributors. This document outlines some of the conventions on development workflow, contributor's notes, contributing steps, contact points and other resources to make your contribution to the projects felicitous.
+
+**Contributor's Notes**
+
+There are few things to note when contributing to the KKP project, which is highlighted below:
+
+
+
+*   KKP project is hosted on Github; thus, Github knowledge is one of the essential pre-requisites 
+*   The KKP documentation is written in markdown language (.md) and located in the [docs repository](https://github.com/kubermatic/docs/tree/master/content/kubermatic)
```

Comment:
_in Markdown_ is enough, no need to mention "language".

Reply:
Done